packages/d3fc-axis/src/axisOrdinal.js

```diff
@@ -37,10 +40,117 @@ const axisOrdinal = (orient, scale) => {
         return { offset: [0, base.tickPadding()] };
     };
 
+    const decorateRotation = s => {

```

Comment:
this logic is currently specific to ordinal axes, it would be good to have the option to rotate linear / continuous axes